preditor/scintilla/documenteditor.py

```diff
@@ -830,11 +846,39 @@ def findTextNotFound(self, text):
 
     def keyPressEvent(self, event):
         key = event.key()
+        modifiers = event.modifiers()
+
+        retPressed = key == Qt.Key_Return
+        altPressed = modifiers == Qt.AltModifier
+        altReturnPressed = altPressed and retPressed
+
         if key == Qt.Key_Backtab:
             self.unindentSelection()
         elif key == Qt.Key_Escape:
             # Using QShortcut for Escape did not seem to work.
             self.showAutoComplete(True)
+        elif altReturnPressed:
+            # If Alt Return pressed, create new unindented line
```

Comment:
I like this feature. I was going to ask you to update the readme about workbox shortcuts, but this is only implemented for the PyQt based QScintilla editor so it's not a feature that is universal. There isn't really a way to communicate that this hidden feature exists. I wonder if we should add a breakdown of the workbox plugin specific features at some point.